shared/teams/team/invites/index.js

```diff
@@ -0,0 +1,63 @@
+// @flow
```

Comment:
All of this is the same, the row processing was just moved out from `render` -> `mergeProps`